source/UIAHandler/utils.py

```diff
@@ -362,3 +363,11 @@ def _shouldSelectivelyRegister() -> bool:
 		return False
 	else:
 		return winVersion.getWinVer() >= winVersion.WIN11_22H2
+
+
+def _shouldUseWtNotifications() -> bool:

```

Comment:
I don't think Windows Terminal is obvious enough to abbreviate as `wT`

```suggestion
def _shouldUseWindowsTerminalNotifications() -> bool:
```